src/mindlessgen/prog/config.py

```diff
@@ -178,8 +178,9 @@ def __init__(self: GenerateConfig) -> None:
         self._increase_scaling_factor: float = 1.3
         self._element_composition: dict[int, tuple[int | None, int | None]] = {}
         self._forbidden_elements: list[int] | None = None
-        self._scale_vdw_radii: float = 4.0 / 3.0
-        self._scale_minimal_bondlength: float = 0.75
+        self._scale_vdw_radii: float = 1.25
+        self._scale_minimal_bondlength: float = 0.8
```

Comment:
The default values should always be consistent with the "template" `mindlessgen.toml` in the repository.